navigator_engine/common/graph_loader.py

```diff
@@ -201,3 +216,25 @@ def _map_excel_boolean(boolean):
     else:
         raise ValueError('Value {} read from Initial Graph Config is not valid; Only TRUE and FALSE are valid values.'
                          .format(boolean))
+
+
+def _parse_resources(resource_cell):
+    resources = []
+
+    if pd.isnull(resource_cell):
+        return resources
+
+    resource_rows = re.split('\n', resource_cell)

```

Comment:
I'm not sure you need to use regex here?

  ```suggestion
    resource_rows = resource_cell.split('\n')
```